doc-jsdoc/pentaho/config/spec/IRuleSelector.jsdoc

```diff
@@ -0,0 +1,74 @@
+/*!
+ * Copyright 2010 - 2016 Pentaho Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * The `config.spec.IRuleSelector` interface describes
+ * the criteria map that determines if a rule is selected
+ * for a given type and context variables.
+ *
+ * It is the type of the [select]{@link pentaho.config.spec.IRule#select} property
+ * of a configuration rule.
+ *
+ * Besides the [type]{@link pentaho.config.spec.IRuleSelector#type} property,
+ * this interface allows filtering on any of the [context properties]{@link pentaho.spec.IContextVars}.
+ *
+ * @name pentaho.config.spec.IRuleSelector
```

Comment:
Please note "selection" to "selector" change in the rename from `IConfigurationRuleSelection` to `IRuleSelector`.